schema/article.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import positron from '../lib/loaders/positron';
+import cached from './fields/cached';
+import AuthorType from './author';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+} from 'graphql';
+
+let ArticleType = new GraphQLObjectType({
+  name: 'Article',
+  fields: () => ({
+    cached: cached,
```

Comment:
Right now the only option for turning off caching is to just use a normal get request instead of the loader:
```js
import positron from '../lib/apis/positron'
// instead of 
import positron from '../lib/loaders/positron'
```

Though really we need to add the ability to turn off caching from within a query and be able to still use the loader to gain memoization.